### PR TITLE
State that in DebugLine the end column can be equal to the start column

### PR DESCRIPTION
**Pull request by Naghasan**
GLSL can in some cases generate this pattern.

Note: the document was regenerated using asciidoctor instead of asciidoc, hence the fixes in 3 tables.

### CODE REVIEW

File: nonsemantic/NonSemantic.Shader.DebugInfo.100.asciidoc [hunk 1]
```diff
@@ -1486,7 +1486,7 @@ instruction. +
  column number where the location begins. +
  +
  'Column End' is the '<id>' of a 32-bit integer *OpConstant* denoting the source
- column number where the location ends. This must be greater than 'Column Start'. +
+ column number where the location ends. This must be greater then or equal to 'Column Start'. +
```
Comment (dgkoch) [inline nonsemantic/NonSemantic.Shader.DebugInfo.100.asciidoc:1489]:
```suggestion
 column number where the location ends. This must be greater than or equal to 'Column Start'. +
```

Comment (Naghasan) [reply nonsemantic/NonSemantic.Shader.DebugInfo.100.asciidoc:1489]: oh my ... thanks

**Review by dgkoch — APPROVED** [review]

**Comment by alan-baker** [comment]
I notice you changed the generator from asciidoc to asciidoctor. That's not necessarily wrong, but it might be odd for people familiar with the previous versions of the spec. In my mind we should maintain the style as much as possible, but maybe it's not worth it.

**Comment by bashbaug** [comment]
Merging as discussed in the August 21st teleconference.

**Pull request merged**